src/jit/flowgraph.cpp

```diff
@@ -21584,7 +21584,6 @@ Compiler::fgWalkResult      Compiler::fgUpdateInlineReturnExpressionPlaceHolder(
                                        : NO_CLASS_HANDLE;
 #endif // defined(FEATURE_HFA) || defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
 
-
```

Comment:
(Minor) Since the edit is just a blank line deletion, this file could be excluded from PR.